frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -41,7 +41,7 @@ export type ObjectReference = {
 export type ObjectMetadata = {
   annotations?: { [key: string]: string };
   clusterName?: string;
-  creationTimestamp?: string;
+  creationTimestamp?: string | number | Date;
```

Comment:
Confused by this.  `creationTimestamp` is a `string` per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata